internal/core/services/tests/schema_test.go

```diff
@@ -23,7 +23,7 @@ func TestSchema_ImportSchema(t *testing.T) {
 	const description = "someDescription"
 	const urlLD = "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/kyc-v3.json-ld"
 	const schemaType = "KYCCountryOfResidenceCredential"
-	const did = "did:iden3:polygon:mumbai:wyFiV4w71QgWPn6bYLsZoysFay66gKtVa9kfu6yMZ"
+	const did = "did:iden3:optimism:sepolia:wyFiV4w71QgWPn6bYLsZoysFay66gKtVa9kfu6yMZ"
```

Comment:
opid method might have been missed?

Reply:
Actually spotted iden3 method on multiple places

Reply:
Mmm yeah, did it on purpose as iden3 is the standard and otherwise it implied an extra setup, but I can try to remove it )

Reply:
Actually ended up adding the setup for other fixes so looks like it worked right away. Removed remaining iden3. Thank you!